lib/ReactViews/Map/Panels/setting-panel.scss

```diff
@@ -26,6 +26,9 @@
 .btn--viewer {
   composes: btn from '../../../Sass/common/_buttons.scss';
   composes: btn-grey from '../../../Sass/common/_buttons.scss';
+
+  color: $text-light;
```

Comment:
Unfortunately I don't think we'll ever be sure of that - I'm pretty sure a lot of these are actually just fixing stuff I missed too :(.